android/app/src/main/java/com/cleverraven/cataclysmdda/SplashScreen.java

```diff
@@ -303,11 +304,13 @@ protected Boolean doInBackground(Void... params) {
                 deleteRecursive(assetManager, externalFilesDir, new File(externalFilesDir + "/data"));
                 deleteRecursive(assetManager, externalFilesDir, new File(externalFilesDir + "/gfx"));
                 deleteRecursive(assetManager, externalFilesDir, new File(externalFilesDir + "/lang"));
-
+                deleteRecursive(assetManager, externalFilesDir, new File(externalFilesDir + "/config"));
```

Comment:
> This will not have a disruptive impact on the game itself.

I am pretty sure it does have disruptive impact - existing `config` folder will be unconditionally deleted during upgrade from a different version and it won't be replaced with anything as we do not bundle `config` folder by default.

Reply:
OK. What I want to confirm is that no matter how I change it, will it not merge? Depending on the answer I will stop continuing

Reply:
> > This will not have a disruptive impact on the game itself.
> 
> I am pretty sure it does have disruptive impact - existing `config` folder will be unconditionally deleted during upgrade from a different version and it won't be replaced with anything as we do not bundle `config` folder by default.

OK. What I want to confirm is that no matter how I change it, will it not merge? Depending on the answer I will stop continuing or you can close then.